<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-DF9ER86" ignored="2" name="Default test(failed)" tests="2" failures="2" timestamp="03 Jan 2023 14:20:37 GMT" time="0.127" errors="0">
  <testcase name="t1" time="0.031" classname="com.testng.Rerun">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testng.Rerun.t1(Rerun.java:9)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- t1 -->
  <testcase name="t3" time="0.005" classname="com.testng.Rerun">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testng.Rerun.t3(Rerun.java:20)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- t3 -->
  <testcase name="t4" time="0.0" classname="com.testng.Rerun">
    <ignored/>
  </testcase> <!-- t4 -->
  <testcase name="t2" time="0.0" classname="com.testng.Rerun">
    <ignored/>
  </testcase> <!-- t2 -->
</testsuite> <!-- Default test(failed) -->
